src/main/java/com/example/web/repository/UserMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.example.web.repository;
+
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+
+import com.example.web.entity.UserModel;
+
+@Mapper
+public interface UserMapper {
+
+	//loginUserテーブルから、usernameが一致しているものを検索
+    @Select("SELECT * FROM loginUser WHERE username = #{username}")
+    public UserModel findByUsername(String username);
```

Comment:
usernameからユーザーが取得できないときに返り値のUserModelはnullになりますよね？
返り値をOptionalに変更してみてサービスで適切にエラーとすることができるとなおよいですね！
```suggestion
    public Optional<UserModel> findByUsername(String username);
```